pkg/riff/commands/function.go

```diff
@@ -114,7 +114,9 @@ func FunctionCreate(buildpackBuilder core.Builder, fcTool *core.Client) *cobra.C
 					return err
 				}
 			} else {
+				_, _ = fmt.Fprintf(cmd.OutOrStdout(), "\nriff function create: built image %s for service %s.\n", f.Spec.RunLatest.Configuration.RevisionTemplate.Spec.Container.Image, f.Name)
```

Comment:
should we really hardcode `riff` here?

Reply:
oops, merged too hastily. A fix is here: https://github.com/projectriff/riff/pull/1206.

Reply:
Good catch @trisberg !